publications-2024.bib

```diff
@@ -1132,6 +1155,17 @@ @Article{2024:lei.piani.ea:weighted
   preprint = {https://arxiv.org/abs/2211.02508}
 }
 
+@Article{2024:li.brune.ea:from,
+  author  = {Li, Kai and Brune, Sascha and Erd{\"o}s, Zolt{\'a}n and Neuharth, Derek and Mohn, Geoffroy and Glerum, Anne},
+  title   = {From Orogeny to Rifting: The Role of Inherited Structures During the Formation of the South China Sea},
+  journal = {Journal of Geophysical Research: Solid Earth},
+  year    = 2024,
+  volume  = 129,
+  number  = 12,
+  pages   = {e2024JB029006},
```

Comment:
Is it that what you want to put for page number? It's the same number as the DOI

Reply:
Sounds good. I just wanted to make sure that it wasn't an oversight